cmd/thanos/query.go

```diff
@@ -141,6 +144,17 @@ func registerQuery(app *extkingpin.App) {
 			return errors.Errorf("Address %s is duplicated for --rule flag.", dup)
 		}
 
+		// Check if the YAML configuration of request.logging is correct. Exit early if error.
```

Comment:
Be careful with comments, comment only surprises. Comments tends to get obsolete quickly.

```suggestion
		// Check if the YAML configuration of request.logging is correct.
```

Reply:
Sure :+1: